Stripe/StripeiOSTests/STPSetupIntentConfirmParamsTest.swift

```diff
@@ -46,7 +46,7 @@ class STPSetupIntentConfirmParamsTest: XCTestCase {
         // card type should have no default mandateData
         XCTAssertNil(params.mandateData)
 
-        for type in ["sepa_debit", "au_becs_debit", "bacs_debit"] {
+        for type in ["sepa_debit", "au_becs_debit", "bacs_debit", "bancontact", "ideal", "eps", "sofort", "link", "us_bank_account", "cashapp", "paypal", "revolut_pay", "klarna"] {
```

Comment:
Shouldn't we only add `.klarna` here?

Reply:
Technically yes, but we fell behind and didn't update this test for other LPMs that were added here, so I figured I'd add them.

https://github.com/stripe/stripe-ios/blob/cf640b0e8c65b36b3ce01020acba8ba37288c5f0/StripePayments/StripePayments/Source/API%20Bindings/Models/SetupIntents/STPSetupIntentConfirmParams.swift#L71